specification/2.0/schema/accessor.schema.json

```diff
@@ -39,10 +65,33 @@
             "gltf_detailedDescription": "The number of attributes referenced by this accessor, not to be confused with the number of bytes or number of components."
         },
         "type": {
-            "type": "string",
             "description": "Specifies if the attribute is a scalar, vector, or matrix.",
-            "enum": [ "SCALAR", "VEC2", "VEC3", "VEC4", "MAT2", "MAT3", "MAT4" ],
-            "gltf_detailedDescription": "Specifies if the attribute is a scalar, vector, or matrix, and the number of elements in the vector or matrix."
+            "anyOf": [
+                {
+                    "enum": [ "SCALAR" ]
+                },
+                {
+                    "enum": [ "VEC2" ]
+                },
+                {
+                    "enum": [ "VEC3" ]
+                },
+                {
+                    "enum": [ "VEC4" ]
+                },
+                {
+                    "enum": [ "MAT2" ]
+                },
+                {
+                    "enum": [ "MAT3" ]
+                },
+                {
+                    "enum": [ "MAT4" ]
+                },
+                {
+                    "type": "string"
```

Comment:
There are a couple of ways these schemas will get used;
- Viewers/Loaders can validate a file before trying to display it. The additional objects in anyOf allow loaders to validate a future version and continue to load it while ignoring any new elements.  
- Validators like in vs-code or glTF-validator may want to ensure the file is exactly 2.0 and not 2.1. Loosening up the schema will not be useful for this type of use case. If strict validations are required then these tools will need to use a stricter version of the schema that is specific to a version and doesn't do the forwards compatibility. 

Do we need to consider making a "strict" version of the schema? By default the schema should be flexible because its still valid 2.0 with the forwards compat capabilities. We could do some script to create the stricter version that specifically checks for 2.0 without forwards compat.